main.go

```diff
@@ -191,6 +208,14 @@ func main() {
 	setupLog.Info(fmt.Sprintf("Go Version: %s", runtime.Version()))
 	setupLog.Info(fmt.Sprintf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH))
 
+	ctx := context.Background()
+	kubeInformerFactory.Start(ctx.Done())
```

Comment:
Instead of using a new `context`, what about using `ctrl.SetupSignalHandler()`?
WDYT @zroubalik ?